samples/multiple-specs/twitter2.api

```diff
@@ -21,6 +21,36 @@ service {
     response 'SearchResponse'
   }
 
+  describe "tweets search" spec {
+    def m = delegate
+    ['#androiddev', 'stanfy'].each { query ->
+
+      m.describe "searching $query" spec {
```

Comment:
Have to figure out how to properly access delegate from here (without this hack `describe` is invoke on service object instead of the nested delegate)...